test/CMakeLists.txt

```diff
@@ -4,11 +4,18 @@ set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
 set(BUILD_MOCK ON CACHE BOOL "" FORCE)
 set(CMAKE_POLICY_DEFAULT_CMP0048 NEW)
 
-add_subdirectory(
-  "${CMAKE_CURRENT_SOURCE_DIR}/gtest-1.10.0"
-  "${CMAKE_CURRENT_BINARY_DIR}/prefix")
-
-include_directories(SYSTEM "${CMAKE_CURRENT_SOURCE_DIR}/gtest-1.10.0/googletest/include")
+find_package(GTest)
+if (NOT GTEST_FOUND)
+    add_subdirectory(
+      "${CMAKE_CURRENT_SOURCE_DIR}/gtest-1.10.0"
+      "${CMAKE_CURRENT_BINARY_DIR}/prefix")
+    include_directories(SYSTEM "${CMAKE_CURRENT_SOURCE_DIR}/gtest-1.10.0/googletest/include")
+else()
+    add_subdirectory(
+      "/usr/src/googletest"
+      "${CMAKE_CURRENT_BINARY_DIR}/prefix")
+    include_directories(SYSTEM "/usr/src/googletest/googlemock/include")
```

Comment:
These paths are wrong, gtest is usually not installed in this location. You should use pkgconf to get the correct paths.

Reply:
Hello, the path is not exported via cmake, this is for custom build in places where the .so dynamic library is not provided.